### DISCUSSION TITLE
Chrome is unresponsive for a while with development version of Tailwind CSS (but is OK with the production one)

### DISCUSSION THREAD

**Discussion by tmcdos** [tailwindlabs/tailwindcss#3296] [Help]
Yesterday I noticed a strange behavior on my project. 
I have a static HTML page with 2 CSS files - the default Tailwind CSS + my own custom styles. 
If I open the HTML file in Firefox 72 - the page works as expected.
But if I open the HTML file in Chrome 79 - the browser enters some heavy loop of style recalculations and layout recomputation and exits the loop after about 90 seconds. Then, when I try to open the single drop-down list on the page - it takes forever and some parts of the browser UI become unresponsive.
However, there is something strange - when I use the production version of the Tailwind CSS file (output from the PurgeCSS), there is no issue and the page works as expected.

To demonstrate the issue:
1) Extract the attached [test.zip](https://github.com/tailwindlabs/tailwindcss/files/5777013/test.zip)
2) Open the `index.html` file (notice that initially it is referencing `tailwind.css`)
3) Now edit `index.html` to reference `_tailwind.css` (notice the leading underscore) and reload the page
4) You should observe the described issue

Any ideas or suggestions how to debug the issue ?

**Comment by Williamssonn** [comment]
Can someone figure out why it goes unresponsive after some time?  if you open Developer Tools in advance you will be able to hit the "Pause" button any  Creating a Chrome Extension with React and TypeScript  I made a collection of more than 120 TailwindCSS blocks ready to copy-paste and is 100%.


[www.mcdvoice.com](https://www.mcdvoice.onl/)
